src/api/public/apidocs-new/paths/source_project_name_cmd_copy.yaml

```
@@ -0,0 +1,55 @@
+post:
+  summary: Copy the entire project.
+  description: |
+    The `copy` command copies the entire project from the source project provided by `oproject` to the target project provided by `project`.
+
+    The params `makeolder` and `makeoriginolder` update the relevant vrev.
+    The param `withbinaries` ensures we copy also the binaries.
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+    - in: query
+      name: oproject
+      schema:
+        type: string
+      required: true
+      description: Origin project name
+      example: "home:Admin"
+    - in: query
+      name: opackage
+      schema:
+        type: string
+      required: true
+      description: Origin package name
+      example: "ctris"
+    - in: query
+      name: makeolder
+      schema:
+        type: boolean
+      required: false
```

```suggestion
      required: false
```
All `required: false` lines could be removed. `required: false` is the default option. It ocurrs in this line and two more lines.